libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SFSDKWebViewStateManager.m

```diff
@@ -71,64 +70,32 @@ + (void)setSessionCookieManagementDisabled:(BOOL)sessionCookieManagementDisabled
     _sessionCookieManagementDisabled = sessionCookieManagementDisabled;
 }
 
-
-+(BOOL) isSessionCookieManagementDisabled {
++ (BOOL)isSessionCookieManagementDisabled {
     return _sessionCookieManagementDisabled;
 }
 
 
 #pragma mark Private helper methods
 
-+ (void)removeWKWebViewCookies:(NSArray *)domainNames withCompletion:(nullable void(^)(void))completionBlock {
++ (void)removeWKWebViewCookies {
     if (_sessionCookieManagementDisabled) {
         [SFSDKCoreLogger d:self format:@"[%@ %@]: Cookie Management disabled. Will do nothing.", NSStringFromClass(self), NSStringFromSelector(_cmd)];
         return;
     }
     
-    NSAssert(domainNames != nil && [domainNames count] > 0, ERR_NO_DOMAIN_NAMES);
     WKWebsiteDataStore *dataStore = [WKWebsiteDataStore defaultDataStore];
     NSSet *websiteDataTypes = [NSSet setWithArray:@[ WKWebsiteDataTypeCookies]];
-    [dataStore fetchDataRecordsOfTypes:websiteDataTypes
-                     completionHandler:^(NSArray<WKWebsiteDataRecord *> *records) {
-                         NSMutableArray<WKWebsiteDataRecord *> *deletedRecords = [NSMutableArray new];
-                         for (WKWebsiteDataRecord * record in records) {
-                             // Cookie record display names look like "salesforce.com", "force.com". Make
-                             // them look like proper cookie domain suffixes, for comparison.
-                             NSString *recordDisplayName = [NSString stringWithFormat:@".%@", record.displayName];
-                             for(NSString *domainName in domainNames) {
-                                 if ([domainName hasSuffix:recordDisplayName]) {
-                                     [deletedRecords addObject:record];
-                                 }
-                             }
-                         }
-                         if (deletedRecords.count > 0) {
-                             [dataStore removeDataOfTypes:websiteDataTypes
-                                           forDataRecords:deletedRecords
-                                        completionHandler:^{
-                                            if (completionBlock)
-                                                completionBlock();
-                                        }];
-                         } else {
-                             dispatch_async(dispatch_get_main_queue(), ^{
-                                 if (completionBlock) {
-                                     completionBlock();
-                                 }
-                             });
-                         }
-                     }];
-}
-
-+ (NSArray<NSString *> *) domains {
-    return @[@".salesforce.com", @".force.com", @".cloudforce.com"];
+    [dataStore removeDataOfTypes:websiteDataTypes modifiedSince:[NSDate distantPast] completionHandler:^{
+        // Intentionally blank because completion can't be nil
```

Comment:
nothing was passing in a completionHandler,  but setting it to nil here generates a warning, can change it back if anyone has a preference